apps/studio/components/interfaces/App/CommandMenu/ApiKeys.tsx

```diff
@@ -23,14 +23,16 @@ export function useApiKeysCommands() {
   const setPage = useSetPage()
 
   const project = useSelectedProject()
-  const { data: settings } = useProjectApiQuery(
+  const { data: settings } = useProjectSettingsV2Query(
     { projectRef: project?.ref },
     { enabled: !!project }
   )
   const ref = project?.ref || '_'
 
-  const anonKey = settings?.autoApiService?.defaultApiKey ?? undefined
-  const serviceKey = settings?.autoApiService?.serviceApiKey ?? undefined
+  const anonKey = (settings?.service_api_keys ?? []).find((x) => x.tags === 'anon')?.api_key
+  const serviceKey = (settings?.service_api_keys ?? []).find(
+    (x) => x.tags === 'service_role'
+  )?.api_key
```

Comment:
I'm seeing this a lot in this PR... worth a cheeky function? Something like:
```typescript
function getDefaultAPIKeys(settings?: ProjectSettings) {
  const anonKey = (settings?.service_api_keys ?? []).find((x) => x.tags === 'anon')?.api_key
  const serviceKey = (settings?.service_api_keys ?? []).find(
    (x) => x.tags === 'service_role'
  )?.api_key

  return { anonKey, serviceKey }
}
```

wdyt?

Reply:
okay just pushed a fix! 🙏